js/ui/pivot_grid/ui.pivot_grid.field_chooser.js

```diff
@@ -334,21 +335,26 @@ var FieldChooser = BaseFieldChooser.inherit({
             that._renderArea($col2, "column");
             that._renderArea($col2, "data");
         } else {
+            $container.addClass("dx-l-2");
             this._renderArea($container, "all");
+            var $layout2Container = $(DIV).addClass("dx-layout-2").appendTo($container);
 
-            $col1 = $(DIV).addClass("dx-col").appendTo($container);
-            $col2 = $(DIV).addClass("dx-col").appendTo($container);
+            $col1 = $(DIV).addClass("dx-col").appendTo($layout2Container);
+            $col2 = $(DIV).addClass("dx-col").appendTo($layout2Container);
 
             that._renderArea($col1, "filter");
             that._renderArea($col1, "row");
             that._renderArea($col2, "column");
             that._renderArea($col2, "data");
         }
-        that.renderSortable();
 
-        that.updateDimensions();
+        if(hasWindow) {
```

Comment:
Why don't move it to the `render` method?

Reply:
Good idea! Will fix it..